diff --git a/node_modules/react-native-enriched/android/src/main/new_arch/react/renderer/components/RNEnrichedTextInputViewSpec/conversions.h b/node_modules/react-native-enriched/android/src/main/new_arch/react/renderer/components/RNEnrichedTextInputViewSpec/conversions.h
index 3669ddd..b43446e 100644
--- a/node_modules/react-native-enriched/android/src/main/new_arch/react/renderer/components/RNEnrichedTextInputViewSpec/conversions.h
+++ b/node_modules/react-native-enriched/android/src/main/new_arch/react/renderer/components/RNEnrichedTextInputViewSpec/conversions.h
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <folly/dynamic.h>
-#include <react/renderer/components/FBReactNativeSpec/Props.h>
+#include <react/renderer/components/rncore/Props.h>
 #include <react/renderer/components/RNEnrichedTextInputViewSpec/Props.h>
 #include <react/renderer/core/propsConversions.h>
 
